File: tools/bench/langs/fib.cs
using System;

class Program {
  static int Fib(int n) {
    if (n < 2) return n;
    return Fib(n - 1) + Fib(n - 2);
  }

  static void Main() {
    Console.WriteLine(Fib(28));
  }
}

File: tools/bench/langs/fib.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func fib(n int64) int64 {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	fmt.Println(fib(size()))
}

func size() int64 {
	n, _ := strconv.ParseInt(os.Args[1], 10, 64)
	return n
}
